tools/merge_conv_bn.py

```diff
@@ -0,0 +1,72 @@
+import argparse
+
+import torch
+import torch.nn as nn
+from mmcv.runner import save_checkpoint
+
+from mmdet.apis import init_detector
+
+
+def merge_conv_bn(conv, bn):
+    conv_w = conv.weight
+    conv_b = conv.bias if conv.bias is not None else torch.zeros_like(
+        bn.running_mean)
+
+    factor = bn.weight / torch.sqrt(bn.running_var + bn.eps)
+    merged_conv_w = conv_w * factor.reshape([conv.out_channels, 1, 1, 1])
+    merged_conv_b = (conv_b - bn.running_mean) * factor + bn.bias
+
+    merged_conv = nn.Conv2d(
+        conv.in_channels,
+        conv.out_channels,
+        conv.kernel_size,
+        conv.stride,
+        conv.padding,
+        bias=True)
```

Comment:
No need to construct a new conv layer, just set the weight and bias of the original conv.